src/visit_vtk/lightweight/vtkUnstructuredGridBoundaryFilter.C

```diff
@@ -1077,16 +1077,16 @@ BQuad::AssignNodes(const vtkIdType *n)
 // ****************************************************************************
 
 void
-BQuad::OutputCell(int node0, vtkPolyData *pd, vtkCellData *in_cd,
+BQuad::OutputCell(vtkIdType node0, vtkPolyData *pd, vtkCellData *in_cd,
                  vtkCellData *out_cd)
 {
     vtkIdType n[4];
-    vtkIdType *list = quad_reorder_list[ordering_case];
-    n[0] = (list[0] == -1 ? node0 : nodes[list[0]]);
-    n[1] = (list[1] == -1 ? node0 : nodes[list[1]]);
-    n[2] = (list[2] == -1 ? node0 : nodes[list[2]]);
-    n[3] = (list[3] == -1 ? node0 : nodes[list[3]]);
-    int newId = pd->InsertNextCell(VTK_QUAD, 4, n);
+    vtkIdType *blist = quad_reorder_list[ordering_case];
```

Comment:
What was driving the change in name from `list` to `blist` here? Is `list` also a class data member name?

Reply:
Yes, there is a `list` class member.